daprdocs/content/en/reference/components-reference/supported-secret-stores/openbao.md

```diff
@@ -0,0 +1,17 @@
+---
+type: docs
+title: 'OpenBao'
+linkTitle: 'OpenBao'
+description: 'Detailed information on the OpenBao secret store component.'
+aliases:
+  - '/operations/components/setup-secret-store/supported-secret-stores/openbao/'
+---
+
+## Usage
+
+Currently, there is no dedicated OpenBao Secrets Store. However, you can utilize
+the `secretstores.hashicorp.vault` component, which has been tested and
```

Comment:
Are *all* the Hashicorp Vault component meta data fields relevant for the OpenBao component or only a subset? If it's only a subset of metadata fields I recommend copying what is relevant to this page.